src/styles/index.scss

```diff
@@ -1,3 +1,7 @@
-body {
-  margin: 0;
-}
+@import '../styles/utils/mixins';
+@import '../styles/utils/variables';
+@import '../styles/block/catalog--card-stule';
```

Comment:
The filename 'catalog--card-stule.scss' seems to be a typo. It should probably be 'catalog--card-style.scss'. Please correct the filename to match the naming convention and ensure consistency.